package.json

```diff
@@ -26,7 +26,7 @@
     "/README.md"
   ],
   "scripts": {
-    "dev": "nodemon",
+    "fix-android": "(cd android; ./format-java.sh)",
```

Comment:
Was there a reason to remove nodemon? If not, we should keep it, it was pretty useful